api/net/Areas/Subscriber/Controllers/ContentController.cs

```diff
@@ -77,11 +77,12 @@ public async Task<IActionResult> FindWithElasticsearchAsync()
         var uri = new Uri(this.Request.GetDisplayUrl());
         var query = Microsoft.AspNetCore.WebUtilities.QueryHelpers.ParseQuery(uri.Query);
         var filter = new ContentFilter(query);
+        var useUnpublished = filter.UseUnpublished == true;
         if (filter.Actions.Any(x => x.Contains("Commentary")) && filter.Quantity == 10)
         {
             filter.Quantity = 500;
         }
-        var result = await _contentService.FindWithElasticsearchAsync(_elasticOptions.PublishedIndex, filter);
+        var result = await _contentService.FindWithElasticsearchAsync(useUnpublished ? _elasticOptions.UnpublishedIndex :_elasticOptions.PublishedIndex, filter);
```

Comment:
@Fosol - just pinging to let you know this has been updated. Also wanted to see if this approach was ok?